docs/content/getting-started/using-pfelements.md

```diff
@@ -0,0 +1,179 @@
++++
+title = "Using PatternFly Elements"
+description = ""
+weight = 3
+draft = false
+bref = ""
+toc = true
+menu = "start"
+tags = [ "start" ]
++++
+
+
+# Using PFE Web Components in your project
+
+
+### **General**
+
+Check out the demo files within each component for examples of the attributes and possible variable overrides.  The readme files within the component directories should have descriptions about the variables for each component. 
+
+The list below is the recommended approach to using web components and overriding style properties, in order.
+
+
+## 1. Markup
+
+
+### **1a. Content Components**
+
+The beauty of web components is that they have much of the styling built-into the tag itself. Start with the tags first. **And please use named slots! **
+
+
+```
+
+      <pfe-card>
+        <h2 slot="pfe-card--header">Light card</h2>
+  <pfe-cta>hello</pfe-cta>
+      </pfe-card>
+```
+
+
+
+### **1b. Container components (see 
+
+<p id="gdcalert7" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: undefined internal link (link text: "container"). Did you generate a TOC? </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert8">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
```

Comment:
Did you mean to commit this?

Reply:
No; its junk from the gdocs export. Removed